### ISSUE TITLE
CPU usage with merge

### ISSUE THREAD

**Issue by nagendraK6** [t-mullen/video-stream-merger#56]
I am trying to use your SDK for stream merging. My first impression is this is really amazing and exactly doing what I need. I see one issue with high cpu usage. Is this expected?

**Comment by t-mullen** [comment]
Yes, you can expect significant CPU usage when merging lots of streams at a high resolution.

You can try the experimental WebGL branch and see if that helps:
https://github.com/t-mullen/video-stream-merger/blob/webgl/dist/video-stream-merger.js
```javascript
const merger = new VideoStreamMerger({ webgl: true })
```

**Comment by nagendraK6** [comment]
I have just 2 streams. One self and other remote user (like a 1:1 video chat). Without merge CPU usage 30-40% with merge goes to 50-60% and varies.

Does this webgl flat work itself or any specific version needs to install?

I have earlier installed with npm install video-stream-merger


now for webgl I am trying 
npm install git://github.com/t-mullen/video-stream-merger/tree/webgl.git

**Comment by Arman92** [comment]
I tested the webgl version, but the performance was worse than normal canvas.
I have a 1070 Ti gpu with Ryzen 3700X. I also tested it with an old HP laptop (Geforce 620M and Core i7-3630QM) on both tests, normal canvas implementation was much better (could output full-HD with at least 5 full-hd sreams merged together, 30 fps, although with high cpu usage)

Webgl implementation only outputed about 5-10 fps with 1070 Ti and 1 frame per 3-5 seconds on my old laptop.

Is there something I'm missing or configured wrong?
